pbe/breakupCoalescenceTables.py

```diff
@@ -0,0 +1,51 @@
+import fluid
+import numpy as np
+import matplotlib.pyplot as plt
+
+path = 'validationData/tables/1/'
+f = open(path + 'parameters', 'w')
+f_br = open(path + 'breakup', 'w')
+f_c = open(path + 'coalescence', 'w')
+
+F = fluid.fluid('coulaloglou', 0, 0)
+F.C1 = 1.52e-02
+F.C2 = 6.78e-02
+F.C3 = 1.06e-12
+F.C4 = 5.13e13
```

Comment:
I don't see why you couldn't do

```python
for n in range(4):
    f.write('C{0}'.format(n + 1) + repr(F.C[n]) + '\n')
```
You don't have to change to `Cbr` and `Ccoal` as you seem to be suggesting. The clarity of the code in kernel function will stay exactly the same apart from using `C[0]` instead of `C1`, which could be solved too.